src/app/shared/item/item-versions/item-versions.component.ts

```diff
@@ -375,6 +376,37 @@ export class ItemVersionsComponent implements OnInit {
     return this.authorizationService.isAuthorized(FeatureID.CanEditVersion, version.self);
   }
 
+  /**
+   * Show submitter in version history table
+   */
+  showSubmitter() {
+
+    const includeSubmitter$ = this.configurationService.findByPropertyName('versioning.item.history.include.submitter').pipe(
+      getFirstSucceededRemoteDataPayload(),
+      map((configurationProperty) => configurationProperty.values[0]),
+      startWith(false),
+    );
+
+    const isAdmin$ = combineLatest([
+      this.authorizationService.isAuthorized(FeatureID.IsCollectionAdmin),
+      this.authorizationService.isAuthorized(FeatureID.IsCommunityAdmin),
+      this.authorizationService.isAuthorized(FeatureID.AdministratorOf),
+    ]).pipe(
+      map(([isCollectionAdmin, isCommunityAdmin, isSiteAdmin]) => {
+        return isCollectionAdmin || isCommunityAdmin || isSiteAdmin;
+      }),
+      take(1),
+      tap((res) => { console.log('isAdmin = ' + res); })
```

Comment:
Can we remove this logging statement?  This seems like it was for debugging purposes.